write-fonts/src/tables/gvar.rs

```diff
@@ -934,4 +969,120 @@ mod tests {
             Some(PackedPointNumbers::Some(vec![1, 3]))
         );
     }
+
+    fn make_39_bytes_of_variation_data() -> Vec<GlyphDeltas> {
```

Comment:
Why does it matter that it's 39 bytes?

Reply:
guess because it's % 2 != 0 so must be 2-byte aligned